service-base/src/test/java/org/eclipse/hono/service/registration/RegistrationServiceTests.java

```diff
@@ -61,6 +61,10 @@
      * The gateway identifier used in the tests.
      */
     protected static final String GW = "gw-1";
+    /**
```

Comment:
I guess that you by mistake included the for tests here instead of the other PR for tests.

Reply:
Yes, indeed. Removed